services/rollout-service/pkg/versions/versions.go

```diff
@@ -218,21 +234,53 @@ func (v *versionClient) ConsumeEvents(ctx context.Context, processor VersionEven
 							continue
 						}
 
-						processor.ProcessKuberpultEvent(ctx, KuberpultEvent{
-							Application:      app.Name,
-							Environment:      env.Name,
-							EnvironmentGroup: envGroup.EnvironmentGroupName,
-							Team:             tm,
-							IsProduction:     env.Priority == api.Priority_PROD,
-							Version: &VersionInfo{
-								Version:        app.Version,
-								SourceCommitId: sc,
-								DeployedAt:     dt,
-							},
-						})
+						if v.manageArgoAppsEnabled && len(v.manageArgoAppsFilter) > 0 && strings.Contains(v.manageArgoAppsFilter, app.Name) {
```

Comment:
The simple string filter here is probably a bit too simple. Also, I think this would be much better in the ArgoProcessor.